include/client/libcurve.h

```diff
@@ -138,7 +138,8 @@ int Create(const char* filename,
  */
 int Create2(const char* filename,
            const C_UserInfo_t* userinfo,
-           size_t size, uint64_t stripeUnit, uint64_t stripeCount);
+           size_t size, uint64_t stripeUnit,
```

Comment:
This function is an external API, if we modify it, we may break the user's code.

We can add another function like `Create3`(with `poolsetName` parameter), and also keep the old `Create2`(without `poolsetName` parameter).

You should only modify these two functions, other functions like `FileClient::Create2` should keep it is.

Reply:
> This function is an external API, if we modify it, we may break the user's code.
> 
> We can add another function like `Create3`(with `poolsetName` parameter), and also keep the old `Create2`(without `poolsetName` parameter).
> 
> You should only modify these two functions, other functions like `FileClient::Create2` should keep it is.

fix